lib/challenge.js

```diff
@@ -58,7 +83,17 @@ var some_pet;
 //    the values of `name`, `species`, `breed`, and `noise`. The constructor function should also give each pet a method called `makeNoise`,
 //    which behaves as described in Q3.
 
-var Pet;
+var Pet = function(name, species, breed, noise) {
+  this.name = name;
+  this.species = species;
+  this.breed = breed;
+  this.noise = noise;
+}
```

Comment:
This is a good constructor function you've made here and assigned to `Pet`. We wanted you to add a prototype method on `Pet` so all of its instances will have it. Half credit.